### DISCUSSION TITLE
kodi compatible trailer naming

### DISCUSSION THREAD

**Discussion by bullwinkl** [RandomNinjaAtk/docker-radarr-extended#39] [General]
Forgive me if it's obvious, but I have kodi compatible trailer naming enabled and it's literally naming all the trailers that are downloaded "movie-trailer.mkv" instead of moviefilename-trailer.mkv.  Is this intentional?  Is this controlled in a setting I haven't found yet?

Thanks in advance!

**Comment by RandomNinjaAtk** [comment]
Yes it's intentional and from my understanding, it should be compatible. But I don't personally use kodi, so it's largely untested...

**Comment by bullwinkl** [comment]
I looked through the documentation at https://kodi.wiki/view/Trailers and I don't see anything about "movie-trailer.ext" being compatible.  Just so I'm clear, it's calling every trailer "movie-trailer.mkv".  All of them, no matter the title of the movie.

From the kodi docs:
`Trailers are named the same as the movie file but have -trailer appended to the name with no spaces`

So if you have a movie file named "Hoosiers (1986).mkv"  a Kodi compatible trailer filename would he "Hoosiers (1986)-trailer.mkv"

Not sure if I'm making sense...but thanks for the quick response!
